projects/pt1/python/torch_mlir/jit_ir_importer/build_tools/torch_ods_gen.py

```diff
@@ -502,6 +502,12 @@ def emit_with_mutating_variants(key, **kwargs):
     emit(
         "aten::_log_softmax : (Tensor, int, bool) -> (Tensor)"
     )
+    emit(
+        "aten::__interpolate.scale_list : (Tensor, int?, float[]?, str, bool?, bool?, bool) -> (Tensor)"
+    )
+    emit(
+        "aten::__interpolate.size_list_scale_list : (Tensor, int[]?, float[]?, str, bool?, bool?, bool) -> (Tensor)"
+    )
```

Comment:
Please add the shape info for both of these ops. Refer https://github.com/llvm/torch-mlir/blob/main/docs/adding_abstract_interpretation_functions.md

Reply:
ok

Reply:
I think there's some confusion here. I'm asking to add the shape info here.